solidjs-tailwind/src/components/RepoCard/RepoCard.jsx

```diff
@@ -1,59 +1,48 @@
 import { Link } from '@solidjs/router';
 import { Show, splitProps } from 'solid-js';
 import RepoMeta from '../RepoMeta/RepoMeta';
-import { OcStar2 } from 'solid-icons/oc';
 import { PrivacyBadge } from '../PrivacyBadge';
 
 const RepoCard = (props) => {
   const [local] = splitProps(props, [
     'name',
     'description',
-    'primaryLanguage',
+    'language',
     'stargazerCount',
     'owner',
-    'isProfilePage',
     'updatedAt',
-    'visibility',
+    'forkCount',
+    'isPrivate',
+    'languageColor',
   ]);
 
-  const repoNameWithOwnerLink = () =>
-    local.owner?.login ? `/${local.owner.login}/${local.name || ''}` : '';
-
-  const repoNameWithOwner = () =>
-    `${!local.isProfilePage ? `${local.owner?.login || ''}/` : ''}${
-      local.name || ''
-    }`;
-
   return (
     <div class="px-4 py-4 border-b border-gray-200 first-of-type:border-t flex justify-between flex-wrap md:flex-nowrap gap-x-4">
       <div class="col-span-12 md:col-span-7 text-left">
         <h3 class="mb-4">
-          <Link href={repoNameWithOwnerLink()}>
+          <Link href={`/${local.owner}/${local.name}`}>
             <span class="text-xl text-blue-600 font-semibold hover:underline mr-3">
-              {repoNameWithOwner()}
+              {local.name}
```

Comment:
no please, return to the previous implementation. If something isn't clear, please ask why it is done that way.